x-pack/plugins/stack_alerts/server/alert_types/geo_containment/tests/geo_containment.test.ts

```diff
@@ -298,5 +333,115 @@ describe('geo_containment', () => {
       expect(allActiveEntriesMap).toEqual(currLocationMap);
       expect(testAlertActionArr).toMatchObject(expectedContext);
     });
+
+    it('should generate multiple alerts per entity if found in multiple shapes in interval', () => {
+      const emptyPrevLocationMap = new Map();
+      const currLocationMapWithThreeMore = new Map([...currLocationMap]).set('d', [
```

Comment:
mind making these timestamps different from eachother, so we can get a better sense of how these should be ordered? similar in the other stub

Reply:
Done! They're different days already but agreed that having the exact same time for each is distracting.